client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -30,12 +30,14 @@ const useDisableOutboundRows = (rows?: OutboundRowFragment[]) => {
 export const OutboundShipmentListViewComponent: FC = () => {
   const { mutate: onUpdate } = useOutbound.document.update();
   const t = useTranslation('distribution');
+  const { updateSortQuery, updatePaginationQuery, filter } =
+    useHandleUrlQueryParams('otherPartyName');
   const navigate = useNavigate();
   const modalController = useToggle();
 
-  const { data, isError, isLoading, sort, pagination, filter } =
+  const { data, isError, isLoading, sortBy, page, first, offset } =
```

Comment:
the sort, page, first & offset don't need to be exported from the hook now - the reason for doing that was due to a lack of shared global context ( each hook call received its own version of the params ). It would be better if those items came from the hook call above, and the `useOutbounds` just fetched them from `useHandleUrlQueryParams` without exporting them again

Reply:
Ah yes, sure. So like this? fe6229e70

Reply:
spot on! I didn't know that you could destructure within a destructure like that 😁  nice!
then you can also remove the export of those props from the hook itself

Reply:
Which prop export do you mean? The `queryParams` return needs to be the same shape as the equivalent object in `useQueryParamsStore` (as it's fed to the DB query in `useOutbounds`), and the filter that is returned is different to the "filterBy" inside it, so not sure how to simplify the return structure further.

Reply:
just that the usage of the `useOutbounds` hook is now

```
  const { data, isError, isLoading } = useOutbound.document.list();
```

but the hook is returning all of the queryParams:

```
  return {
    ...useQuery(api.keys.paramList(queryParams), () =>
      api.get.list(queryParams)
    ),
    ...queryParams,
  };
```

it can be simplified down to 

```
  return useQuery(api.keys.paramList(queryParams), () =>
    api.get.list(queryParams)
  );
```

Reply:
Ah right, yes, you mean that hook. Sure, will do.